airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/source.py

```diff
@@ -115,13 +114,12 @@ def convert_config2stream_args(cls, config: Mapping[str, Any]) -> Mapping[str, A
 
     @classmethod
     def convert_config_to_declarative_stream_args(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
-        """Convert input configs to parameters of the future streams
-        This function is used by unit tests too
+        """
+        Sets defaults to `start_date` and `ignore_pagination`.
         """
         return {
             "subdomain": config["subdomain"],
             "start_date": config.get("start_date", cls.get_default_start_date()),
-            "auth_type": config.get("auth_type"),
```

Comment:
I didn't see any reference to `auth_type` in the manifest hence why I removed this